assembly/src/ast/procedure/resolver.rs

```diff
@@ -13,17 +13,7 @@ pub enum ResolvedProcedure {
     /// The name was resolved to a procedure exported from another module
     External(FullyQualifiedProcedureName),
     /// The name was resolved to a procedure with a known MAST root
-    MastRoot(Span<RpoDigest>),
-}
-
-impl Spanned for ResolvedProcedure {
```

Comment:
Why remove the `Spanned` impl here? If you have an `RpoDigest`, but no `SourceSpan`, just construct the `Span<RpoDigest>` with an empty/default span. That is perfectly valid, and allows us to propagate source spans when we have that information available.

Reply:
I reimplemented it, with one difference: `MastRoot` now wraps a `RpoDigest` instead of `Span<RpoDigest>`, and I don't remember if there is a good reason for  that?